src/shortcuts/splitSentences.tsx

```diff
@@ -17,14 +17,7 @@ import splitSentence from '../util/splitSentence'
 
 // eslint-disable-next-line jsdoc/require-jsdoc, react-refresh/only-export-components
 const Icon = ({ fill = 'black', size = 20, style }: IconType) => (
-  <svg
-    version='1.1'
-    width={size}
-    height={size}
-    fill={fill}
-    style={{ ...style, paddingTop: '8px' }}
```

Comment:
splitSentences is the only icon in /icons I believe that doesn't use the .icons class, which is why it needed inline padding. Since I moved the padding to the toolbar, it is vertically centered, minus 2px that I'll fix (see comment above)